apps/namadillo/src/lib/query.ts

```diff
@@ -123,12 +128,12 @@ export const signTxArray = async <T>(
     );
 
     if (!signedTxs) {
-      throw new Error("Signing failed");
+      throw new Error("Signing failed: No signed transactions returned");
```

Comment:
This is fine, just a note though: My Ledger signing PR handles this a bit differently, where we can see the chain ID validation message if there is mismatch, but we can leave this for now!